src/7zip-cpp/7zpp/SevenZipLibrary.cpp

```diff
@@ -2,15 +2,12 @@
 #include "SevenZipLibrary.h"
 #include "GUIDs.h"
 
+STDAPI CreateObject(const GUID *clsid, const GUID *iid, void **outObject);
```

Comment:
I appreciate headers are too much to ask for in CCOM day and age, but it might be worth a comment to explain this is the entry point to 7zip and where it's defined. Or does this just magically happen when you click it in VS? :)